src/inc/CombFilter.h

```diff
@@ -0,0 +1,47 @@
+//
+//  CombFilter.hpp
+//  CombFilter
+//
+//  Created by Vedant Kalbag on 09/02/22.
+//
+
+#ifndef CombFilter_h
+#define CombFilter_h
+
+#include <stdio.h>
+#include "RingBuffer.h"
+
+
+
+class CCombFilterBase
+{
+public:
+//    CCombFilterBase(int delayLength, int iNumChannels, float gain=0.5);
+    CCombFilterBase(int delayLength, int iNumChannels, float gainValue=0.5);
+//    ~CCombFilterBase();
```

Comment:
I think it would be better to just define the destructor here and let the child classes inherit it.

Reply:
Good point, thanks for pointing that out!